packages/guilded.js/lib/structures/CalendarEvent.ts

```diff
@@ -13,187 +13,202 @@ import { Collection } from "@discordjs/collection";
-  /** Custom Id generated for the rsvp */
-  readonly id: string;
-  /** The ID of the calendar event (min 1) */
-  readonly calendarEventId: number;
-  /** The ID of the channel */
-  readonly channelId: string;
-  /** The ID of the server */
-  readonly serverId: string;
-  /** The ID of the user */
-  readonly userId: string;
-  /** The status of the rsvp ("going", "maybe", "declined", "invited", "waitlisted", or "not responded") */
-  status: string;
-  /** The ID of the user who created this rsvp */
-  readonly createdBy: string;
-  /** The ISO 8601 timestamp that the rsvp was created at */
-  readonly _createdAt: number;
-  /** The ID of the user who updated this rsvp */
-  updatedBy?: string | null;
-  /** The ISO 8601 timestamp that the rsvp was updated at, if relevant */
-  updatedAt?: string | null;
-
-  constructor(client: Client, data: CalendarEventRsvpPayload) {
-    super(client, data);
-
-    this.id = data.calendarEventId + "-" + data.userId;
-    this.calendarEventId = data.calendarEventId;
-    this.channelId = data.channelId;
-    this.serverId = data.serverId;
-    this.userId = data.userId;
-    this.status = data.status;
-    this.createdBy = data.createdBy;
-    this._createdAt = Date.parse(data.createdAt);
-    this.updatedBy = null;
-    this.updatedAt = null;
-
-    this._update(data);
-  }
-
-  get author(): User | null {
-    return this.client.users.cache.get(this.createdBy) ?? null;
-  }
-
-  get createdAt(): Date {
-    return new Date(this._createdAt);
-  }
-
-  _update(data: Partial<CalendarEventRsvpPayload>): this {
-    if ("updatedAt" in data && typeof data.updatedAt !== "undefined") {
-      this.updatedAt = data.updatedAt ?? null;
+    /** Custom Id generated for the rsvp */
+    readonly id: string;
+    /** The ID of the calendar event (min 1) */
+    readonly calendarEventId: number;
+    /** The ID of the channel */
+    readonly channelId: string;
+    /** The ID of the server */
+    readonly serverId: string;
+    /** The ID of the user */
+    readonly userId: string;
+    /** The status of the rsvp ("going", "maybe", "declined", "invited", "waitlisted", or "not responded") */
+    status: string;
+    /** The ID of the user who created this rsvp */
+    readonly createdBy: string;
+    /** The ISO 8601 timestamp that the rsvp was created at */
+    readonly _createdAt: number;
+    /** The ID of the user who updated this rsvp */
+    updatedBy?: string | null;
+    /** The ISO 8601 timestamp that the rsvp was updated at, if relevant */
+    updatedAt?: string | null;
+
+    constructor(client: Client, data: CalendarEventRsvpPayload) {
+        super(client, data);
+
+        this.id = data.calendarEventId + "-" + data.userId;
+        this.calendarEventId = data.calendarEventId;
+        this.channelId = data.channelId;
+        this.serverId = data.serverId;
+        this.userId = data.userId;
+        this.status = data.status;
```

Comment:
It's a little redundant to put this in both the constructor and the update method if the constructor calls update.